src/appengine/libs/issue_management/jira/__init__.py

```diff
@@ -113,7 +113,7 @@ def components(self):
     """The issue component list."""
     return self._components
 
-  def save(self):
+  def save(self, new_comment=None, notify=True):  # pylint: disable=unused-argument
```

Comment:
Add "FIXME: Add support for new_comment and notify arguments".

Reply:
Added, please check.